HISTORY.md

```diff
@@ -1,5 +1,12 @@
 ### ensmallen ?.??.?: "???"
 ###### ????-??-??
+ * Correction in the formulation of sigma in CMA-ES
+    ([#183](https://github.com/mlpack/ensmallen/pull/183)).
+
+ * Fixing ProgressBar
```

Comment:
Can we be a little more explicit about what got fixed here?  When people read the changelogs, we can probably generally assume they don't read much further.  I might suggest, e.g.,

```
 * Fix total number of epochs and time estimation for ProgressBar callback
    ([#181](https://github.com/mlpack/ensmallen/pull/181)).
```

Reply:
Hi, @rcurtin I have changed this accordingly. Kindly have a look. Thanks